src/main/java/se/bjurr/violations/lib/example/OtherClass.java

```diff
@@ -4,12 +4,15 @@
  * No ending dot
  */
 public class OtherClass {
- public static String CoNstANT = "yes";
+ public static String CoNstANT = "yes"; 
 
  public void myMethod() {
   if (CoNstANT.equals("abc")) {
 
   }
+  if (CoNstANT.equals("abc")) {
```

Comment:
Reporter: PMD
Rule: EmptyIfStmt
Severity: WARN
File: /home/bjerre/workspace/jenkinsci/violation-comments-to-github-plugin/work/workspace/test/violations-test/src/main/java/se/bjurr/violations/lib/example/OtherClass.java L13
 

Empty Code http://pmd.sourceforge.net/pmd-5.2.3/pmd-java/rules/java/empty.html#EmptyIfStmt

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>